### PR TITLE
Slobs chatbot style fixes

### PR DESCRIPTION
**Pull request by Nitsorn**
- change FormWrapper to VFormGroup
- can reset chat alerts
- morgans comments
- upcoming items
- changed metadata.alpha to alphaNum
- removed auto-permitting everyone from protection

### CODE REVIEW

File: app/components/page-components/Chatbot/Modules/ChatbotModule.vue [hunk 1]
```diff
@@ -16,14 +20,16 @@
...
-      <br />
-      <br />
       <div class="chatbot-module__action">
         <button
+          v-if="!chatbotModule.comingSoon"
           @click="chatbotModule.onExpand"
           class="button button--action"
         >
-          PREFERENCES
+          Preferences
```
Comment (holiber) [inline app/components/page-components/Chatbot/Modules/ChatbotModule.vue]: $t

File: app/components/page-components/Chatbot/windows/ChatbotCustomCommandWindow.vue.ts [hunk 2]
```diff
@@ -110,13 +113,20 @@ export default class ChatbotCustomCommandWindow extends ChatbotWindowsBase {
...
 
-    this.chatbotApiService.createCustomCommand(this.newCommand);
+    this.chatbotApiService
+      .createCustomCommand(this.newCommand)
+      .catch(this.onErrorHandler);
+  }
+
+  onErrorHandler(errorResponse: IChatbotErrorResponse) {
+    if (errorResponse.error && errorResponse.error === 'Duplicate') {
+      alert('This command is already taken. Try another command.');
```
Comment (holiber) [inline app/components/page-components/Chatbot/windows/ChatbotCustomCommandWindow.vue.ts]: $t

File: app/components/page-components/Chatbot/ChatbotModules.vue.ts [hunk 3]
```diff
@@ -20,6 +20,11 @@ export default class ChatbotModules extends ChatbotBase {
 
   get modules() {
     const backgroundUrlSuffix = this.nightMode ? 'night' : 'day';
+    const comingSoonText = $t(`
+      Streamlabs is diligently working on the next release of Chatbot.
+       Stay tuned.
+       We have more features on the way.
+    `);
```
Comment (gettinToasty) [inline app/components/page-components/Chatbot/ChatbotModules.vue.ts]: i don't think you can use template literals with the `$t` function, should use single quote strings instead

File: app/components/shared/inputs/ToggleInput.vue [hunk 4]
```diff
@@ -40,8 +40,8 @@
   .transition();
   position: absolute;
   z-index: 2;
```
Comment (holiber) [inline app/components/shared/inputs/ToggleInput.vue]: Do we really need z-index for these inputs?

File: app/components/page-components/Chatbot/ChatbotModules.vue.ts (hunk 3 above)
Comment (Nitsorn) [reply app/components/page-components/Chatbot/ChatbotModules.vue.ts]: oh interesting. Thanks @gettinToasty

**Pull request merged**